server/store/wish.go

```diff
@@ -0,0 +1,5 @@
+package store
+
+type Wish struct {
```

Comment:
Most of the things in store feel that would make more sense in a "model" package, since they feel more definitions of the objects than information about how to store them.

Reply:
I thought about it, not quite ready for it; my thinking was - call it `model`, and define a KV interface in the package, with SaveToKV/LoadFromKV for the relevant structs. Separate PR?